Tests/UnitTests/Application/MainCoordinatorTests.swift

```diff
@@ -124,6 +148,8 @@ extension MainCoordinatorTests {
         XCTAssertTrue(sut.childCoordinators[3] is LoginCoordinator)
         // THEN the root's delegate is the MainCoordinator
         XCTAssertTrue(sut.root.delegate === sut)
+
+        UserDefaults.standard.removeObject(forKey: FeatureFlags.enableWalletVisibleToAll.rawValue)
```

Comment:
As this line seems to be used in most of the tests, should it be included in the teardown?

Reply:
In the new PR we're actually using:
```swift
AppEnvironment.updateReleaseFlags([:])
```
in tests to reset things which is a slightly nice way than calling `UserDefaults` directly